index.html

```diff
@@ -857,111 +872,94 @@ <h2>
         <p>
           This section is a work in progress as this document evolves.
         </p>
-        <p>
-          The documents listed below outline various privacy considerations for
-          Digital Credentials, both broadly and for presentation on the web.
-          Their contents will be integrated into this document gradually.
-        </p>
-        <ul>
-          <li>
-            <a href=
-            "https://github.com/WICG/digital-credentials/blob/main/horizontal-reviews/security-privacy.md">
-            TAG Security and Privacy Considerations Questionnaire (WIP)</a>
-          </li>
-          <li>
-            <a href="https://github.com/w3cping/credential-considerations">W3C
-            Privacy Interest Group (PING) Credentials Considerations
-            repository</a> which contains in-progress list of concerns and
-            implications of identity-verification and credential presentation,
-            for consideration in Internet and Web standardization
-          </li>
-          <li>
-            <a href=
-            "https://github.com/w3c-cg/threat-modeling/blob/main/models/decentralized-identities.md">
-            Threat Model for Decentralized Identities</a>
-          </li>
-        </ul>
       </div>
-      <section>
-        <h3>
-          Unnecessary Requests for Credentials
-        </h3>
-        <p class="issue" title="Work in progress">
-          Explain how the API could be used to unnecessarily request digital
-          credentials from individuals such as requesting a driver's license to
-          log into a movie rating website and how the ecosystem can mitigate
-          this risk.
-        </p>
-      </section>
-      <section>
-        <h3>
-          Over Collection of Data
-        </h3>
-        <p class="issue" title="Work in progress">
-          Explain how the API could be used to request more data than necessary
-          for a transaction and how the ecosystem can mitigate that over
-          collection.
-        </p>
-      </section>
-      <section>
-        <h3>
-          Individual Consent
```

Comment:
Consent seems important and germane. There isn't consensus in the group yet on whether consent is gathered by the site, the browser or the wallet, and that desperately needs to be documented. Where consent will be obtained determines how information flows between the parties. (If consent is handled by the wallet, then all the context inforemation needs to be passed along to the wallet, for example.)